components/calendly/invitee-canceled.js

```diff
@@ -0,0 +1,41 @@
+const calendly = require("https://github.com/PipedreamHQ/pipedream/components/calendly/calendly.app.js");
+
+module.exports = {
+  name: "Invitee Cancelled (Instant)",
+  description: "Emits an event when an invitee cancels a scheduled event",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    calendly,
+    db: "$.service.db",
+    http: "$.interface.http",
```

Comment:
We made some recent changes to the HTTP interface. When you need a custom response like you do below, you'll need to set 

```javascript
http: {
  type: "$.interface.http",
  customResponse: true,
}
```